app/_hub/kong-inc/ai-proxy-advanced/overview/_index.md

```diff
@@ -0,0 +1,199 @@
+---
+nav_title: Overview
+---
+
+The AI Proxy Advanced plugin lets you transform and proxy requests to multiple AI providers and models at the same time. 
+This lets you set up load balancing between targets.
+
+The plugin accepts requests in one of a few defined and standardised formats, translates them to the configured target format, and then transforms the response back into a standard format.
+
+The following table describes which providers and requests the AI Proxy Advanced plugin supports:
+
+| Provider | Chat | Completion | Streaming |
+| -------- | ---- | ---------- | --------- |
+| OpenAI (GPT-4, GPT-3.5) | ✅ | ✅ | ✅ |
+| OpenAI (GPT-4o and Multi-Modal) | ✅ | ✅ | ✅ |
+| Cohere | ✅ | ✅ | ✅ |
+| Azure | ✅ | ✅ | ✅ |
+| Anthropic | ✅ | ❌ | Only chat type |
+| Mistral (mistral.ai, OpenAI, raw, and OLLAMA formats) | ✅ | ✅ | ✅ |
+| Llama2 (raw, OLLAMA, and OpenAI formats) | ✅ | ✅ | ✅ |
+| Llama3 (OLLAMA and OpenAI formats) | ✅ | ✅ | ✅ |
+
+## How it works
+
+The AI Proxy Advanced plugin will mediate the following for you:
+
+* Request and response formats appropriate for the configured `provider` and `route_type`
+* The following service request coordinates (unless the model is self-hosted):
+  * Protocol
+  * Host name
+  * Port
+  * Path
+  * HTTP method
+* Authentication on behalf of the Kong API consumer
+* Decorating the request with parameters from the `config.options` block, appropriate for the chosen provider
+* Recording of usage statistics of the configured LLM provider and model into your selected [Kong log](/hub/?category=logging) plugin output
+* Optionally, additionally recording all post-transformation request and response messages from users, to and from the configured LLM
+* Fulfillment of requests to self-hosted models, based on select supported format transformations
+
+Flattening all of the provider formats allows you to standardize the manipulation of the data before and after transmission. It also allows your to provide a choice of LLMs to the Kong consumers, using consistent request and response formats, regardless of the backend provider or model.
+
+This plugin currently only supports REST-based full text responses.
+
+## Load balancing
+
+This plugin supports following load balancing algorithms:
+* Lowest-usage 
+* [Round-robin (weighted)](/gateway/latest/how-kong-works/load-balancing/#round-robin)
+* [Consistent-hashing (sticky-session on given header value)](/gateway/latest/how-kong-works/load-balancing/#consistent-hashing)
+
+## Semantic routing
+
+Info about semantic routing?
```

Comment:
This plugin supports distributing requests based on similarity of prompts to the given description of each model, allowing Kong to select specific model that performs better
in given domains.